guides/doc-Content_Management_Guide/topics/Managing_Activation_Keys.adoc

```diff
@@ -255,32 +255,8 @@ Configure Red Hat Subscription Manager to use {ProjectServer} for registration a
 
 To use an activation key for host registration with an existing Red Hat Enterprise Linux 7 host to {ProjectServer}, complete the following steps:
 
-. Download the consumer RPM for your {ProjectServer}.
```

Comment:
I think we should keep this workflow but put it after the global registration one and mark it as deprecated.